tests/integration/data_sources_and_expectations/test_canonical_expectations.py

```diff
@@ -33,6 +33,8 @@
     SqliteDatasourceTestConfig(),
 ]
 
+JUST_PANDAS_DATA_SOURCES: Sequence[DataSourceTestConfig] = [PandasDataFrameDatasourceTestConfig()]
```

Comment:
I think we'll eventually phase out this file and move this and the other constant list out, but for now, this seemed like the easiest place to put this.